nextcloud-jail.sh

```diff
@@ -140,20 +144,29 @@ if [ "${DB_PATH}" = "${FILES_PATH}" ] || [ "${FILES_PATH}" = "${PORTS_PATH}" ] |
 then
   echo "DB_PATH, FILES_PATH, CONFIG_PATH and PORTS_PATH must all be different!"
   exit 1
+elif [ "${THEMES_PATH}" = "${THEMES_PATH}" ] || [ "${THEMES_PATH}" = "${PORTS_PATH}" ] || [ "${THEMES_PATH}" = "${DB_PATH}" ] || [ "${THEMES_PATH}" = "${CONFIG_PATH}" ]
+  echo "DB_PATH, FILES_PATH, CONFIG_PATH and PORTS_PATH must all be different!"
```

Comment:
This error should also mention THEMES_PATH

Reply:
fixed